src/nexus/handlers/data.py

```diff
@@ -136,15 +156,17 @@ def parse_format_line(self, data):
         except IndexError:
             return None
 
-        line = line.lower().replace(" =", "=").replace("= ", "=")  # standardise
+        line = line.replace(" =", "=").replace("= ", "=")  # standardise
         for chunk in WHITESPACE_PATTERN.split(line):
             try:
                 key, value = chunk.split("=", maxsplit=1)
                 value = QUOTED_PATTERN.sub('\\1', value)
             except ValueError:
                 key, value = chunk, True
             if key:
-                out[key] = value
+                key = key.lower()
+                out[key] = value if key in ['missing', 'gap'] else \
+                    (value.lower() if isinstance(value, str) else value)
```

Comment:
@SimonGreenhill so this is ok with you? As you can see in the tests, it makes for a more transparent way of categorizing sites.

Reply:
looks good but I can't think of any circumstances where lower here is necessary.  One the one hand this is the least breaking change, but on the other hand maybe we could just go with: `out[key] = value` ?

Reply:
I tried this first, but that breaks a test, where
```python
assert f['datatype'] == 'rna'
```

Reply:
hmm. ok. again, I can't think why we'd want to lower case that. Maybe we should fix the test? what do you think?

Reply:
I think, I'd fix the test. The spec enumerates datatypes in uppercase (or small caps ...see below). But it probably is less confusing, when the library reads verbatim what's in the file - rather than normalizes according to an unknown standard :)

Reply:
Lowercasing the key OTOH seems useful, so I'd keep this.

Reply:
yeah we don't want to have to check for uppercase/lowercase key